libs/@local/hash-backend-utils/src/pg-tables.ts

```diff
@@ -2,74 +2,106 @@
  * This module contains type definitions for tables in the Postgres database. Column
  * names are converted from snake_case to camelCase for consistency.
  */
+import { SupportedRealtimeTable } from "./realtime";
 import { Wal2JsonMsg } from "./wal2json";
 
-export class Entity {
-  constructor(
-    /**
-     * @todo Update table definition when provenance info is added for updates
-     *   https://app.asana.com/0/1202805690238892/1202848989198291/f
-     */
-    public entityId: string,
-    public version: string,
-    public entityTypeVersionId: string,
-    public properties: Record<string, unknown>,
-    public createdBy: string,
-  ) {}
+export type EntityEdition = {
+  entity_edition_id: string;
```

Comment:
Don't we want to reuse `EntityEditionId` here?

Reply:
Hmm, I think that would be fine to do if I use the branded types now instead. Since these types map the PG tables, I'm not sure if we want to treat them as types given by `hash-subgraph`